client/v2/algod/getProof.go

```diff
@@ -13,6 +13,11 @@ type GetProofParams struct {
 
 	// Format configures whether the response object is JSON or MessagePack encoded.
 	Format string `url:"format,omitempty"`
+
+	// Hashtype the type of hash function used to create the proof, must be one of:
+	// * sha512_256
+	// * sha256
+	Hashtype string `url:"hashtype,omitempty"`
```

Comment:
I'm not sure if changes to this file and `client/v2/common/models/proof_response.go` are ready for release, and it looks like they'll be addressed in #309 anyway. So I'd suggest removing them from this PR for now.